src/navigation/Tab/tab.stories.tsx

```diff
@@ -28,22 +28,7 @@ export const SimpleTab = (): React.ReactElement => {
         selectedTab={selected}
         variant="outlined"
         items={items}
-      />
-      {selected}
-    </>
-  );
-};
-
-export const TabContained = (): React.ReactElement => {
```

Comment:
is it possible to omit this story from snapshots instead of removing it? I think it's important to have a story with all the possible cases for each component. 

If not, don't worry, it's fine for now.

Reply:
In the [doc](https://www.npmjs.com/package/@storybook/addon-storyshots), there is an option called `storyNameRegex`, sorry for not sharing the link seems like they do not handle anchors.

Reply:
another option, could be this one https://github.com/storybookjs/storybook/tree/master/addons/storyshots/storyshots-core#storykindregex. We could add an omit or similar to the file name.

Reply:
Cool, thanks! I'll restore the story 👌